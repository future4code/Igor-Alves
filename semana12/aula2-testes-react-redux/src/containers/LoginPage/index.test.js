import React from 'react';
import renderer from 'react-test-renderer';
import { LoginPage } from './index'

it('Snapshot LoginPage', ()=> {
    const tree = renderer
    .create(<LoginPage></LoginPage>)
    .toJSON();
    expect(tree).toMatchSnapshot()
})