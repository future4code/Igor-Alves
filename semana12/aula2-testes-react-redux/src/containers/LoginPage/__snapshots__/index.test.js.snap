// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot LoginPage 1`] = `
<div>
  <div
    className="sc-AykKE dYwhLR"
  >
    <header
      className="sc-AykKC dRUEqp"
    >
      <img
        alt="Logo 4eddit"
        className="sc-AykKD eSgZQA"
        src="logoheader.jpg"
      />
      <button
        className="sc-fzXfMz hWgpqs"
      >
        Cadastre-se
      </button>
    </header>
  </div>
  <form
    className="sc-fzXfNL dItTxs"
  >
    <div
      className="sc-fzXfNO eGYQTF"
    >
      <label>
        E-mail 
      </label>
      <input
        className="sc-fzXfNM zkwOK"
        name="email"
        onChange={[Function]}
        type="email"
        value=""
      />
    </div>
    <div
      className="sc-fzXfNO eGYQTF"
    >
      <label>
        Senha
      </label>
      <input
        className="sc-fzXfNM zkwOK"
        name="password"
        onChange={[Function]}
        type="password"
        value=""
      />
    </div>
    <button
      className="sc-fzXfNN cRawYd"
      onClick={[Function]}
      type="submit"
    >
      Login
    </button>
  </form>
  <footer
    className="sc-fzXfNK codHpx"
  >
    4eddit powered by future4
  </footer>
</div>
`;
